python/README.md

```diff
@@ -96,17 +96,17 @@ You can also download and generate global gridded canopy-app inputs using Python
 
 2. Global inputs require a lot of memory, suggest not running on head node:
 
-   Slurm batch script suggestion:
+   Slurm batch script suggestion (cpu time = ~20-30 min for processing a single time step):
    ```
    #!/bin/bash -l
    #SBATCH --partition=bigmem             # big memory node
    #SBATCH --job-name=canopy-app          # name the job
    #SBATCH --output=canopy-%j.out         # write stdout to named file
    #SBATCH --error=canopy-%j.err          # write stdout to named file
-   #SBATCH --time=0-04:00:00              # Run for max of 00 hrs, 10 mins, 00 secs
+   #SBATCH --time=0-02:00:00              # Run for max of 00 hrs, 10 mins, 00 secs
    #SBATCH --nodes=1                      # Request N nodes
    #SBATCH --ntasks=1                     # Request n tasks
-   #SBATCH --mem-per-cpu=1000GB           # Request nGB RAM per core
+   #SBATCH --mem-per-cpu=60GB             # Request nGB RAM per core
```

Comment:
@angehung5   A single node/task with your suggestion of 60GB memory may not work, please test this example.  

I don't think we really need to increase ntasks because its not parallel code.

From my experience using a single node/task requires like about 256 GB memory.